esx_service/vmdk_ops_test.py

```diff
@@ -731,7 +740,7 @@ def setUp(self):
                 self.datastore_path = datastore[2]
                 logging.debug("datastore_name=%s datastore_path=%s", self.datastore_name,
                                                                      self.datastore_path)
-                if len(datastores) >= 1:
+                if len(datastores) >1:
```

Comment:
Ran the vmdk_ops_test.py test only with one datastore - vmfs. Results are as follows:
================= 

Running unit tests in /tmp/vmdk_ops_unittest3474/vmdk_ops_test.py...
WARNING:root:dockvols creation failed - Operation not permitted on local.3-0
WARNING:root:dockvols creation failed - Operation not permitted on local.3-1
ss........sss....
----------------------------------------------------------------------
Ran 17 tests in 204.492s

OK (skipped=5)